### ISSUE TITLE
Zotero.item_creator_types(itemtype)

### ISSUE THREAD

**Issue by retorquere** [edtechhub/zotero-cli#55]
Returns a dict of all valid creator types for the specified item type

**Comment by bjohas** [comment]
I don't really understand what this means.

**Comment by retorquere** [comment]
You wrote it, I just imported the content of the list you made as issues.

**Comment by bjohas** [comment]
The list came from the pyzotero documentation. I had assumed that most of these were more or less direct calls to the Zotero API, but they may not be. 

It would have been more sensible of is just looked at the zot api spec. Let me investigate that on the next day or so.

**Comment by retorquere** [comment]
Yeah I shouldn't have imported all these issues. Neither of us understands what they mean for the most part and the CLI is going to end up in a mess in a hurry.

Honestly I'd propose we close *all* these issues and you just write a list of things you actually need from the CLI rather than trying to cover everything when we don't have a use-case, or worse, implement a CLI command for everything that's in pyzotero because TBH we don't know the use-case behind most of them, either. It's going to be just busywork this way.
